packages/dds/tree/src/simple-tree/treeNodeApi.ts

```diff
@@ -101,11 +102,19 @@ export interface TreeNodeApi {
 	readonly status: (node: TreeNode) => TreeStatus;
 
 	/**
-	 * Returns the {@link FieldKind.Identifier | identifier} of the given node in the most compressed form possible.
+	 * Returns the {@link SchemaFactory.identifier | identifier} of the given node in the most compressed form possible.
 	 * @remarks
-	 * If the node's identifier is a valid `StableNodeKey`, then this will return a unique process-local integer corresponding to that identifier.
-	 * If the node's identifier is any other string, then this will return that string.
-	 * If the node has no identifier (that is, it has no field of an {@link FieldKind.Identifier | identifier} field kind), then this returns undefined.
+	 * If the node's identifier is a valid UUID that was automatically generated by the SharedTree, then this will return a unique process-local integer corresponding to that identifier.
```

Comment:
```suggestion
	 * If the node's identifier is a valid UUID that was automatically generated by the SharedTree, then this will return a document-unique integer corresponding to that identifier.
```

Reply:
Also, maybe worth clarifying that process-unique means unique within the instance of the container on the client.

Reply:
There's a clause below that sort of says that but it says "runtime" rather than "container" - I'll change it to "container" since that's a more concrete thing that devs interact with.